discord/guild.py

```diff
@@ -119,7 +119,7 @@ class Guild(Hashable):
     """
 
     __slots__ = ('afk_timeout', 'afk_channel', '_members', '_channels', 'icon',
-                 'name', 'id', 'unavailable', 'name', 'region', '_state',
+                 'name', 'description', 'id', 'unavailable', 'name', 'region', '_state',
```

Comment:
Since you're not actually setting `description` anywhere, there is no need for it to be in the `__slots__`. Is this received on `GUILD_CREATE` by any chance? If so you can add an attribute and document it.

Reply:
From what I can tell, there is nothing sent over `GUILD_CREATE`. I'll remove it, thank you.